src/webrtc/call.ts

```diff
@@ -2257,7 +2272,21 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
                 this.hangup(CallErrorCode.IceFailed, false);
             }
         } else if (this.peerConn?.iceConnectionState == "disconnected") {
-            this.iceDisconnectedTimeout = setTimeout(() => {
+            this.candidatesEnded = false;
+            // noinspection TypeScriptValidateTypes
```

Comment:
What's the invalid type here?

Reply:
Always the same my IDE recognizes the node js method here. I turned it off because it was handicapping me. I forgot to remove the comments.